### PR TITLE
feat: create model programm #35

### PR DESCRIPTION
**Pull request by AndreLZGava**

### CODE REVIEW

File: src/database/migration/1637764982139-create_programm.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class createProgramm1637764982139 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'programm',
```
Comment (danilolutz) [inline src/database/migration/1637764982139-create_programm.ts:7]: "program" has only one "m"

File: src/database/migration/1637764982139-create_programm.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class createProgramm1637764982139 implements MigrationInterface {
```
Comment (danilolutz) [inline src/database/migration/1637764982139-create_programm.ts:3]: "program" has only one "m"

File: src/database/migration/1637764982139-create_programm.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            type: 'varchar',
+          },
+        ],
+      }),
+      true
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.dropTable('programm');
```
Comment (danilolutz) [inline src/database/migration/1637764982139-create_programm.ts:29]: "program" has only one "m"

File: src/electron/Main.ts [hunk 4]
```diff
@@ -17,6 +17,7 @@ import { Book } from './database/models/book.schema';
 import { User } from './database/models/user.schema';
 import { Event } from '../electron/contracts/Event';
 import Maker from './database/factory/maker';
+import { Programm } from './database/models/programm.schema';
```
Comment (danilolutz) [inline src/electron/Main.ts:20]: "program" has only one "m"

File: src/electron/database/models/programm.schema.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class Programm
```
Comment (danilolutz) [inline src/electron/database/models/programm.schema.ts:4]: "program" has only one "m"

**Review by danilolutz — CHANGES_REQUESTED** [review]

File: src/electron/database/models/programm.schema.ts (hunk 5 above)
Comment (AndreLZGava) [reply src/electron/database/models/programm.schema.ts]: done

File: src/electron/Main.ts (hunk 4 above)
Comment (AndreLZGava) [reply src/electron/Main.ts]: done

File: src/database/migration/1637764982139-create_programm.ts (hunk 3 above)
Comment (AndreLZGava) [reply src/database/migration/1637764982139-create_programm.ts]: done

File: src/database/migration/1637764982139-create_programm.ts (hunk 2 above)
Comment (AndreLZGava) [reply src/database/migration/1637764982139-create_programm.ts]: done

File: src/database/migration/1637764982139-create_programm.ts (hunk 1 above)
Comment (AndreLZGava) [reply src/database/migration/1637764982139-create_programm.ts]: done

**Review by danilolutz — APPROVED** [review]

**Pull request merged**